src/navigator.js

```diff
@@ -26,6 +26,14 @@ class Navigator {
         }
       }
 
+      const enableHighAccuracy = Storage.getItem(Storage.ENABLE_HIGH_ACCURACY);
```

Comment:
should this default to `true`? looks like it might be `false` if not specified

Reply:
Probably makes sense to preserve the existing behavior (default to false).